File: java/easy/(111) longest-word.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            String longestWord = "";
            String[] sentance = line.split(" ", -1);

            for (String word : sentance){
                if (word.length() > longestWord.length()){
                    longestWord = word;
                }
            }

            System.out.println(longestWord);
        }
    }
}

File: python/easy/(111) longest-word.py
import sys

def main(filepath):
    with open(filepath, 'r') as f:
        for line in f.readlines():
            if line:
                line = line.strip()
                line = line.split(' ')
                
                longest_word = ''
                for word in line:
                    if len(word) > len(longest_word):
                        longest_word = word

                print longest_word


if __name__ == '__main__':
    main(sys.argv[1])
